examples/transactions/examples/gas_price_usd.rs

```diff
@@ -0,0 +1,64 @@
+//! Example of how to get the gas price in USD using the Chainlink ETH/USD feed.
+
+use alloy_network::Ethereum;
+use alloy_node_bindings::{Anvil, AnvilInstance};
+use alloy_primitives::{address, utils::format_units, Address, Bytes, U256};
+use alloy_provider::{HttpProvider, Provider};
+use alloy_rpc_client::RpcClient;
+use alloy_rpc_types::TransactionRequest;
+use alloy_sol_types::{sol, SolCall};
+use alloy_transport_http::Http;
+use eyre::Result;
+use reqwest::Client;
+use std::str::FromStr;
+
+const ETH_USD_FEED: Address = address!("5f4eC3Df9cbd43714FE2740f5E3616155c5b8419");
+const ETH_DECIMALS: u32 = 18;
+
+sol!(
+    #[derive(Debug)]
+    function latestAnswer() external view returns (int256);
+);
+
+#[tokio::main]
+async fn main() -> Result<()> {
+    let (provider, _anvil) = init();
+
+    let call = latestAnswerCall {}.abi_encode();
+    let input = Bytes::from(call);
+
+    let tx = TransactionRequest {
+        to: Some(ETH_USD_FEED),
+        input: Some(input).into(),
+        ..Default::default()
+    };
```

Comment:
Show this working with the transaction builder methods instead of setting up the struct